docs/rules/jsx-no-literals.md

```diff
@@ -21,12 +21,15 @@ var Hello = <div>{'test'}</div>;
 
 ### Options
 
-There is only one option:
+There are two options:
 
 * `noStrings` - Enforces no string literals used as children, wrapped or unwrapped.
+* `validateProps` - Enforces no template literals used as props.
```

Comment:
why only template? i'd expect *any* literals to be detected by this option

Reply:
I was trying to mimic the same behaviour as before, will detect string literals too.